stories/containers/pay.stories.js

```diff
@@ -55,100 +35,100 @@ const store = new Store({
   },
 
   isProcessing: false
-})
+}
 
 const mockPayInvoice = async () => {
   action('mockPayInvoice')
-  store.set({ isProcessing: true })
-  await delay(2000)
-  store.set({ isProcessing: false })
+  // data.set({ isProcessing: true })
+  // await delay(2000)
+  // data.set({ isProcessing: false })
 }
 
 const mockSendCoins = async () => {
   action('mockSendCoins')
-  store.set({ isProcessing: true })
-  await delay(2000)
-  store.set({ isProcessing: false })
+  // data.set({ isProcessing: true })
+  // await delay(2000)
+  // data.set({ isProcessing: false })
 }
 
 const mockQueryFees = async () => {
   action('mockQueryFees')
-  store.set({ isQueryingFees: true })
-  await delay(2000)
-  store.set({
-    onchainFees: {
-      fastestFee: 8,
-      halfHourFee: 8,
-      hourFee: 4
-    }
-  })
-  store.set({ isQueryingFees: false })
+  // data.set({ isQueryingFees: true })
+  // await delay(2000)
+  // data.set({
+  //   onchainFees: {
+  //     fastestFee: 8,
+  //     halfHourFee: 8,
+  //     hourFee: 4
+  //   }
+  // })
+  // data.set({ isQueryingFees: false })
 }
 
 const mockQueryRoutes = async pubKey => {
   action('mockQueryRoutes', pubKey)
-  store.set({ isQueryingRoutes: true })
-  await delay(2000)
-  const nodes = store.get('nodes')
-  if (nodes.find(n => n.pub_key === pubKey)) {
-    store.set({
-      routes: [
-        {
-          total_time_lock: 547118,
-          total_fees: '0',
-          total_amt: '10000',
-          hops: [
-            {
-              chan_id: '565542601916153857',
-              chan_capacity: '15698',
-              amt_to_forward: '10000',
-              fee: '0',
-              expiry: 546974,
-              amt_to_forward_msat: '10000010',
-              fee_msat: '21'
-            }
-          ],
-          total_fees_msat: '21',
-          total_amt_msat: '10000021'
-        },
-        {
-          total_time_lock: 547118,
-          total_fees: '0',
-          total_amt: '10000',
-          hops: [
-            {
-              chan_id: '565542601916153857',
-              chan_capacity: '15698',
-              amt_to_forward: '10000',
-              fee: '0',
-              expiry: 546974,
-              amt_to_forward_msat: '10000010',
-              fee_msat: '3'
-            }
-          ],
-          total_fees_msat: '3',
-          total_amt_msat: '10000021'
-        }
-      ]
-    })
-  } else {
-    store.set({ routes: [] })
-  }
-  store.set({ isQueryingRoutes: false })
+  // data.set({ isQueryingRoutes: true })
```

Comment:
Is this commented code is intended to stay or was forgotten to be removed?

Reply:
Meant to be removed - have updated